GTG/gtk/browser/main_window.py

```diff
@@ -1315,6 +1315,26 @@ def on_dismiss_task(self, widget=None):
                 task.set_status(Task.STA_DISMISSED)
                 self.close_all_task_editors(uid)
 
+    def on_reopen_task(self, widget=None):
+        tasks_uid = [uid for uid in self.get_selected_tasks()
+                     if uid is not None]
+        if len(tasks_uid) == 0:
+            return
```

Comment:
Not needed. `tasks_uid` is an emtpy list, so the following `tasks` and `tasks_status` are empty as well, and `zip([], [], [])` is empty too, so the loop isn't entered.
I know you've copied from above, but if you're already here you can change this instance.